### PR TITLE
Update “lit-html” dependency.

### PR DESCRIPTION
**Pull request by theengineear**

### CODE REVIEW

**Comment by theengineear** [comment]
@klebba — Think we're ready to hop on the `2.0` train?

File: x-element.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import { asyncAppend } from '../../lit-html/directives/async-append.js';
 import { asyncReplace } from '../../lit-html/directives/async-replace.js';
 import { cache } from '../../lit-html/directives/cache.js';
 import { classMap } from '../../lit-html/directives/class-map.js';
-import { directive, html, render, svg } from '../../lit-html/lit-html.js';
```
Comment (theengineear) [inline x-element.js:5]: This was the only thing I had to change. The `directive` export no longer exists inside the `lit-html.js` file.

File: yarn.lock [hunk 2]
```diff
@@ -606,10 +611,12 @@ levn@^0.4.1:
...
-lit-html@^1.2.1:
-  version "1.2.1"
-  resolved "https://registry.yarnpkg.com/lit-html/-/lit-html-1.2.1.tgz#1fb933dc1e2ddc095f60b8086277d4fcd9d62cc8"
-  integrity sha512-GSJHHXMGLZDzTRq59IUfL9FCdAlGfqNp/dEa7k7aBaaWD+JKaCjsAk9KYm2V12ItonVaYx2dprN66Zdm1AuBTQ==
+lit-html@^2.0.0-rc.1:
+  version "2.0.0-rc.2"
+  resolved "https://registry.yarnpkg.com/lit-html/-/lit-html-2.0.0-rc.2.tgz#c7ac652af1df7e5d8e32a3e6130221766d09cbde"
+  integrity sha512-rl3vtIQ0jq6r0GVbg+57Et9ra+iNhiz/v5V7uPTb6VxnjJaCCYKI7WkzKNlyzjMM2N/ytih3Uxb5vyyaOpjb0Q==
+  dependencies:
+    "@types/trusted-types" "^1.0.1"
```
Comment (theengineear) [inline yarn.lock:619]: They also added a dependency for trusted types.

File: package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "/etc"
   ],
   "dependencies": {
-    "lit-html": "^1.2.1"
+    "lit-html": "^2.0.0-rc.1"
```
[the remark this answers is not in the document]
Comment (theengineear) [reply package.json:25]: One more thing — I think, if anything, the material impact would be performance. I think that they've done quite a bit to shed unnecessary code and code-paths. That said, there's really no problem waiting on this one.

**Comment by theengineear** [comment]
We resolved to _not_ merge this yet. We will wait until we get the official™️ `2.0` release to make this update. I'll leave the PR open to keep the context in one thread though 👍

**Comment by theengineear** [comment]
Up to `2.0.0-rc.2` — I'll continue to keep an eye out for the first `2.0.0` release.

**Comment by theengineear** [comment]
@klebba — Ooook. _Now_ I think we're ready. Any reservations/hesitations? This _just_ targets `lit-html` — I plan to update other things (like in dev dependencies) in a separate change set, e.g., updating eslint/puppeteer/etc.

**Comment by theengineear** [comment]
@klebba — Update to the latest version of lit (again), lemme know if you're ok with the change?

Also, the update to `puppeteer` was related to an issue I hit `yarn installing` that super old version. Just bumping to the latest worked (and tests pass).

**Review by klebba — APPROVED** [review]
Nice, thanks!

**Pull request merged**